spec/controllers/searches_controller_spec.rb

```diff
@@ -394,12 +394,13 @@
     before do
       expect(Affiliate).to receive(:find_by_name).and_return(affiliate)
       expect(affiliate).to receive(:force_mobile_format?).and_return(true)
-      get :index, query: 'gov', affiliate: affiliate.name #, m: 'true'
+      get :index, query: 'gov', affiliate: affiliate.name, m: 'true'
```

Comment:
@peggles2 , the `m: 'true'` part needs to be removed entirely. With that present, it makes this spec a false positive, because it the spec is testing that the mobile format is returned if `force_mobile_format?` is `true`, but `m: 'true'` also forces the mobile format. So this line should just be:
```ruby
get :index, query: 'gov', affiliate: affiliate.name
```